storage/innobase/sync/sync0arr.cc

```diff
@@ -1403,8 +1396,6 @@ sync_arr_fill_sys_semphore_waits_table(
 						OK(field_store_ulint(fields[SYS_SEMAPHORE_WAITS_READERS], rw_lock_get_reader_count(rwlock)));
 						OK(field_store_ulint(fields[SYS_SEMAPHORE_WAITS_WAITERS_FLAG], (longlong)rwlock->waiters));
 						OK(field_store_ulint(fields[SYS_SEMAPHORE_WAITS_LOCK_WORD], (longlong)rwlock->lock_word));
-						OK(field_store_string(fields[SYS_SEMAPHORE_WAITS_LAST_READER_FILE], innobase_basename(my_atomic_loadptr_explicit(&rwlock->last_s_file_name, MY_MEMORY_ORDER_RELAXED))));
-						OK(fields[SYS_SEMAPHORE_WAITS_LAST_READER_LINE]->store(my_atomic_load32_explicit(&rwlock->last_s_line, MY_MEMORY_ORDER_RELAXED), true));
 						fields[SYS_SEMAPHORE_WAITS_LAST_READER_LINE]->set_notnull();
```

Comment:
I think that both `SYS_SEMAPHORE_WAITS_LAST_READER_LINE` and `SYS_SEMAPHORE_WAITS_LAST_READER_FILE` and related `INFORMATION_SCHEMA.INFORMATION_SCHEMA.INNODB_SYS_SEMAPHORE_WAITS` table columns must be removed. This removal must be mentioned in the commit message too.